server/.golangci.yml

```diff
@@ -122,7 +122,7 @@ issues:
         channels/app/platform/web_conn.go|\
         channels/app/platform/web_hub.go|\
         channels/app/platform/web_hub_test.go|\
-        channels/app/plugin_api_tests/manual.test_http_hijack_plugin/main.go|\
+        channels/app/plugin_api_test.go|\
```

Comment:
The diff looks wrong. Can you please fix it? Thanks!

Reply:
Apologies, had issue while rebasing. Done